packages/patternfly-4/react-charts/src/components/Chart/Chart.tsx

```diff
@@ -422,26 +419,24 @@ export const Chart: React.FunctionComponent<ChartProps> = ({
       }
     });
 
+    const offsetY = xAxisLabelHeight + legendTitleHeight;
     if (legendPosition === ChartLegendPosition.bottom) {
-      dy += ChartCommonStyles.legend.margin + xAxisLabelHeight + legendTitleHeight;
+      dy += offsetY;
```

Comment:
Sorry, I think I was a little vague in my original comment. I was referring to the fact that we are adding the `offsetY` value to `dy` in both branches of the condition. We can pull `dy` out of this conditional altogether and just say `const dy = xAxisLableHeight + legendTitleHeight`. So then we can define both dy and dx as constants after defining xAxisLabelHeight and legendTitleHeight.

```js
const dx = (legendPosition === ChartLegendPosition.bottomLeft) ? -10 : 0;
const dy = xAxisLabelHeight + legendTitleHeight;
```

Reply:
Oh I see now, it's `else if` not `else`. Disregard then.